app/controllers/users.py

```diff
@@ -786,11 +788,46 @@ def get(self):
             if errors:
                 return {"message": "Validation errors", "errors": errors}, 400
 
-            # check if admin
+            
             if not is_admin(current_user_roles):
                 validated_data_query['user_id'] = current_user_id
 
-            # check for start and end query params
+            # check if owner or admin or member for project, database or app
+            project_id = None
+            if validated_data_query.get('a_project_id'):
+                project_id = validated_data_query.get('a_project_id')
+                validated_data_query.pop('user_id',None)
+                
+            elif validated_data_query.get('a_db_id'):
+                validated_data_query.pop('user_id',None)
```

Comment:
I get the reason for removing the `user_id` from the filters. I guess that's to be able to get logs for all users in a **Project**.
There is a scenarial we are not handling and that is if I want to see only activity for a particular user in a project. Meaning I would have to pass `user_id` that is not mine.

Hence you might need to add the case for:

` If validated_data['user_id'] and user_id != current_user_id:
    // You let the user_id reach the db as query
`